src/components/Card.js

```diff
@@ -10,19 +10,23 @@ class Card extends Component {
         const { name, description, from } = this.props.kudos
         return (
             <StyledCard>
-                <Row>
+                <Row margin={'0 0 8px'}>
                     <h6>
                         <KudosTitle>KUDOS </KudosTitle>
                         for
-                        <KudosMan slackUser={this.isSlackUser(name)}> {name}</KudosMan>
+                        <KudosMan slackUser={this.isSlackUser(name)}> {name[0] === '@' ? name.substring(1, name.length) : name}</KudosMan>
```

Comment:
`name[0] === '@'` dusplicates logic from `Card#isSlackUser`.

Reply:
isSlackUser is deleted :)